reference/paths/card-details.yaml

```
@@ -0,0 +1,40 @@
+---
+operationId: card-details
```

Should it be
```suggestion
operationId: get-card-details
```
for consistency?